techniques/failures/F87.html

```diff
@@ -45,7 +45,7 @@
          <ol>
             <li>Examine all content inserted through use of the :before and :after pseudo-elements and the content property</li>
             <li>Verify that the content is decorative.</li>
-            <li>If the inserted content is not decorative, check that the information is provided to assistive technologies and is also available when CSS is turned off.</li>
+            <li>If the inserted content is not decorative, check that the information is available when the :before and :after styles are overriden, or when CSS is turned off.</li>
```

Comment:
I'm wondering how this evolves. What if the browsers allow stacking of styles so the "jim:" that precedes a line in the can also have a paragraph symbol or series of dots to indicate a heading if the user wants that? Would the "or when CSS is turned off" still make sense?  I get that today the turning off of CSS is the easy substitute for the actual test, but am not sure if it is the right thing to do.